tests/queries/0_stateless/02116_tuple_element.sql

```diff
@@ -14,6 +14,8 @@ EXPLAIN SYNTAX SELECT tupleElement(t1, 2) FROM t_tuple_element;
 SELECT tupleElement(t1, 'a') FROM t_tuple_element;
 EXPLAIN SYNTAX SELECT tupleElement(t1, 'a') FROM t_tuple_element;
 
+SELECT tupleElement(t1, 1) AS t10 FROM t_tuple_element FORMAT TSVWithNames;
```

Comment:
@qieqieplus fix looks valid. We prefer to not modify old tests, better to create new ones. Also could you please add test like `SELECT tupleElement(t1, 1) AS t10, t10 FROM t_tuple_element FORMAT TSVWithNames;`. Just want to be sure that alias will work.

Reply:
Thanks for review, I made changes as requested.